virtcontainers/sandbox.go

```diff
@@ -463,6 +466,8 @@ type Sandbox struct {
 	wg *sync.WaitGroup
 
 	shmSize uint64
+
+	cgroups cgroupsManager
```

Comment:
`cgroups *cgroupsManager`?

Reply:
Yes, I adding this to enclose the implementations and data of cgroups handlings in `cgroups.go` (will be `cgroups_linux.go` and `cgroups_unsupported.go`)